### PR TITLE
DIGI_1_CreateStore

### PR DESCRIPTION
**Pull request by rajendarreddy1234**
1.Added StoreModel
2.Added StoreLibrary
3.Updated LibraryUtility.
4.Updated EndPoint.
5.Added StoreController.
6.Added Create.cshtml

### CODE REVIEW

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/StoreController.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+namespace DigiOrderEnterprise.Controllers
+{
+    public class StoreController : Controller
+    {
+        StoreLibrary storeLibrary = null;
+        // GET: Store
+        public ActionResult Index()
+        {
+            return View();
+        }
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/StoreController.cs:16]: Remove this method

File: DigiOrderEnterprise/DigiOrderEnterprise/Views/Store/Create.cshtml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<div class="login-form-grids">
+    <h5>Create Store</h5>
+
+    @using (Html.BeginForm("Register", "Store", FormMethod.Post))
+     {
+        @Html.TextBoxFor(x => x.TITLE, new { @placeholder = "Title...", @required = "", @id = "txtTitle" })
+        @Html.TextBoxFor(x => x.DETAILS, new { @placeholder = "Details...", @required = "", @id = "txtDetails" })
+        @*<input type="text" placeholder="Title..." required=" " id="txtTitle" value="@Model.TITLE">
+            <input type="text" placeholder="Description..." required="txtDetails" value="@Model.DETAILS">*@
+        //<input type="file" placeholder="Upload Image..." id="imgFile">
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Views/Store/Create.cshtml:16]: Remove commet lines

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/StoreLibrary.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+using System.Net.Http;
+using System.Threading.Tasks;
+
+
+namespace DigiOrderEnterpriseLibrary
+{
+    public class StoreLibrary
+    {
+        LibraryUtility libraryUtility = null;
+        Task<HttpResponseMessage> httpResponse = null;
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/StoreLibrary.cs:12]: Add one line space

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs [hunk 4]
```diff
@@ -51,30 +49,66 @@ public async Task<HttpResponseMessage> GetAPISync(string endPoint)
         /// <returns></returns>
         public async Task<HttpResponseMessage> POSTAsync(string endPoint, string Json)
         {
-
-            string apiUrl = Convert.ToString(ConfigurationManager.AppSettings["digiapiurl"]) + endPoint;
+            Task<HttpResponseMessage> response = null;
+            var val = getTokenDetails();
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs:54]: Remove space line here

File: DigiOrderEnterprise/DigiOrderEnterprise/Views/Store/Create.cshtml (hunk 2 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterprise/Views/Store/Create.cshtml:16]: Removed

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/StoreController.cs [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+{
+    public class StoreController : Controller
+    {
+        StoreLibrary storeLibrary = null;
+
+        //Author: Rajendar Reddy konuganti
+        //7-1-2020
+        //Description: Post a Store to DataBase
+        //Reviewer:Inserting the data from frontend to database though WebApi
+        //Review Version:1.0
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/StoreController.cs:19]: add date and Review name

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/StoreController.cs [hunk 6]
```diff
@@ -0,0 +1,44 @@
+﻿using DigiOrderEnterpriseLibrary;
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/StoreController.cs:1]: Add .csproj for all files

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs (hunk 4 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs]: Resolved

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/StoreLibrary.cs (hunk 3 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterpriseLibrary/StoreLibrary.cs:13]: Resolved

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/StoreController.cs (hunk 6 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/StoreController.cs:1]: Resolved

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/StoreLibrary.cs (hunk 3 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterpriseLibrary/StoreLibrary.cs:13]: Resolved

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj [hunk 7]
```diff
@@ -121,6 +121,8 @@
     <Compile Include="Controllers\HomeController.cs" />
     <Compile Include="Controllers\MenuItemController.cs" />
     <Compile Include="Controllers\TaxPageController.cs" />
+    <Compile Include="Controllers\StoreController.cs" />
+    <Compile Include="Controllers\TaxPageController.cs" />
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj:125]: Remove taxpage contoller.cs

File: DigiOrderEnterprise/DigiOrderEnterprise/Web.config [hunk 8]
```diff
@@ -5,13 +5,13 @@
...
-	  <add key="digiapiurl" value="http://localhost/DigiOrderApi/api/" />
-	  <add key="digiapiuserid" value="ENTERPRISE" />
-	  <add key="digiapipassword" value="B75A5773-E01A-422C-9958-6194D8E9D0F8"/>
+    <add key="webpages:Version" value="3.0.0.0" />
+    <add key="webpages:Enabled" value="false" />
+    <add key="ClientValidationEnabled" value="true" />
+    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
+    <add key="digiapiurl" value="http://localhost/DigiOrderApi/api/" />
+    <add key="digiapiuserid" value="ENTERPRISE" />
+    <add key="digiapipassword" value="B75A5773-E01A-422C-9958-6194D8E9D0F8"/>
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Web.config:14]: Remove Space in config file

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj (hunk 7 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj]: Resolved

File: DigiOrderEnterprise/DigiOrderEnterprise/Web.config (hunk 8 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterprise/Web.config]: Resolved

**Pull request merged**